### DISCUSSION TITLE
require("fzf-lua").files() and quickfix list

### DISCUSSION THREAD

**Discussion by desdic** [ibhagwan/fzf-lua#1559] [Q&A] [answered]
Hi,

I'm not sure if its a bug or intended so I'm asking here. But if I mark multiple files via `files` and use alt-q to add them to my quickfix list they produce a format that `cfdo` can't seem to use.

to reproduce:
1. mark files and press enter and they will be moved to the quickfix list (default options)
2. run `:cfdo edit %` and nothing happens.

is this a bug or intended ?

**Comment by ibhagwan** [comment]
I never tried `cfdo edit %`, are you trying to load all buffers? `cfdo %s/…` for search and replace does work so the format isn’t the issue here.

Can you post a screenshot of your qf just to make sure there isn’t a bug causing your qf list to be malformed?

**Reply by desdic** [reply]
I don't specifically try to open files its just an example :) .. I wanted to run a macro via `cfdo normal @q` but couldn't.  For example with grep I get

![withgrep](https://github.com/user-attachments/assets/2dcfb6dc-fa9a-455d-a195-55e08ce2cd1f)

and cfdo works after greping since it uses same format at vimgrep but for files it looks like this

![withfiles](https://github.com/user-attachments/assets/eb1bf1cd-a982-4e78-926f-e90885265a54)

**Reply by ibhagwan** [reply]
> and cfdo works after greping since it uses same format at vimgrep but for files it looks like this

I guess the qf requires line/col no matter what for this to work, I would consider this a bug, def need fixing.

#1560

**Reply by desdic** [reply]
Thank you very much and thank you for this plugins :)

**Reply by ibhagwan** [reply, marked as answer]
@desdic, should work as expected post https://github.com/ibhagwan/fzf-lua/commit/545e789b461ad6c155b35677ad6933a720e43696.

Ty for the kind words :-)

**Reply by desdic** [reply]
Works perfectly, thank you :)
